packages/@aws-cdk/aws-pipes-alpha/lib/logs.ts

```diff
@@ -36,6 +40,52 @@ export enum LogLevel {
   TRACE = 'TRACE',
 }
 
+/**
+ * Log format for `S3LogDestination` logging configuration.
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-s3logdestination.html#cfn-pipes-pipe-s3logdestination-outputformat
+ */
+export enum S3OutputFormat {
+  /**
+  * PLAIN
+  */
+  PLAIN = 'plain',
+  /**
+  * JSON
+  */
+  JSON = 'json',
+  /**
+  * W3C
+  */
+  W3C = 'w3c',
```

Comment:
It's probably worth giving the full name of the spec here and linking to it, should be [Extended Log File Format](https://www.w3.org/TR/WD-logfile) according to the [docs](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes-create.html)